pkgs/applications/audio/faust/faust2alqt.nix

```diff
@@ -1,15 +1,37 @@
 { faust
 , alsa-lib
-, qt4
+, qtbase
+, writeText
+, makeWrapper
 }:
 
-faust.wrapWithBuildEnv {
+faust.wrapWithBuildEnv rec {
 
   baseName = "faust2alqt";
 
   propagatedBuildInputs = [
     alsa-lib
-    qt4
+    qtbase
   ];
 
+  dontWrapQtApps = true;
+
+  # Wrap the binary coming out of the the compilation script, so it knows QT_PLUGIN_PATH
+  wrapBinary = writeText "wrapBinary" ''
+    source ${makeWrapper}/nix-support/setup-hook
+    for p in $FILES; do
```

Comment:
skimming the script this gets appended to, I saw something like `$BINARIES`. why are we using FILES and not that?

Reply:
``$BINARIES`` has extra characters in it that would have needed to be stripped out.
This was just simpler.